### ISSUE TITLE
Responsive Time Log Diagram

### ISSUE THREAD

**Issue by vasu-gondaliya** [vasu-gondaliya/cpu-scheduling-algorithms#2]
Make the time log diagram to be responsive with screen size.
The arrows and tables need to be placed such a way that it remains same for all screens.

**Comment by johnzarnowski** [comment]
Hi I have a project for a software methodology class to contribute to an open source repository and I'm interested in this project. Could you let me know if you're still interested in getting the issue solved and potentially be assigned to it?

**Comment by vasu-gondaliya** [comment]
Sorry I am not getting what said. Are you saying you want to solve this issue and I assign this issue to you?

**Comment by johnzarnowski** [comment]
Yes; I would love to work on the issue & would appreciate if you could assign it to me (sorry for the confusing phrasing)

**Comment by vasu-gondaliya** [comment]
Sure @johnzarnowski . Go for it. Ask for any help if you want.

**Comment by johnzarnowski** [comment]
@vasu-gondaliya I was able to add a horizontal scrollbar that appears when the window is changed to a size that is smaller than the width of the table. Is that what you were looking for?

**Comment by vasu-gondaliya** [comment]
I am talking about the animation with red arrows and tables below output table. Are you talking about the input table? Or this animation. If you have added the scroll bar for the input table then it's good. I will merge it but this issue is for that animation time log.

**Comment by johnzarnowski** [comment]
Hi @vasu-gondaliya I spent the weekend working on the issue & I couldn't figure out how to solve the issue. I did submit a pull request that added the horizontal scrollbar to the input table. I appreciate all of your help & wish you the best of luck in this & all of your other projects in the future!

**Comment by vasu-gondaliya** [comment]
Thank you for your efforts @johnzarnowski . Do you have any advice on how to solve this issue?

**Comment by johnzarnowski** [comment]
@vasu-gondaliya If you reference my pull request, you could try applying the same change that I made on the input table to the time log diagram

**Comment by vasu-gondaliya** [comment]
@johnzarnowski  There is no pull request made.

**Comment by sr-04** [comment]
issue resolved?

**Comment by shubhamtalan996** [comment]
@vasu-gondaliya  I have added a pull request for this issue.
